src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -119,281 +119,363 @@
+    <comment>The color for "bright yellow" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_CursorColor.Text" xml:space="preserve">
     <value>Cursor color</value>
+    <comment>The color of the cursor displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_Cyan.Header" xml:space="preserve">
     <value>Cyan</value>
+    <comment>The color for "cyan" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_Foreground.Text" xml:space="preserve">
     <value>Foreground</value>
+    <comment>The foreground color for text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_Green.Header" xml:space="preserve">
     <value>Green</value>
+    <comment>The color for "green" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_Purple.Header" xml:space="preserve">
     <value>Purple</value>
+    <comment>The color for "purple" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_SelectionBackground.Text" xml:space="preserve">
     <value>Selection background</value>
+    <comment>The background color for selected text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_Red.Header" xml:space="preserve">
     <value>Red</value>
+    <comment>The color for "red" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_White.Header" xml:space="preserve">
     <value>White</value>
+    <comment>The color for "white" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="ColorScheme_Yellow.Header" xml:space="preserve">
     <value>Yellow</value>
+    <comment>The color for "yellow" text displayed on the screen. Users can select a color to use in this context.</comment>
   </data>
   <data name="Globals_AlwaysShowTabs.Header" xml:space="preserve">
     <value>Always show tabs</value>
+    <comment>Header for a control to toggle if the app should always show the tabs (similar to a website browser).</comment>
   </data>
   <data name="Globals_AlwaysShowTabs.HelpText" xml:space="preserve">
     <value>When unchecked, the tab bar will appear when a new tab is created.</value>
-  </data>
-  <data name="Globals_CopyFormatting.Content" xml:space="preserve">
-    <value>Copy formatting</value>
-  </data>
-  <data name="Globals_CopyFormatting.HelpText" xml:space="preserve">
-    <value>When checked, the color and font formatting of selected text is also copied to your clipboard. When unchecked, only plain text is copied to your clipboard.</value>

```

Comment:
unused?

Reply:
Yup! I think at one point we thought this was going to be a toggle. But instead we just created the radio button approach.